### PR TITLE
wait until uptime advances when testing stats

### PR DESCRIPTION
**Pull request by MartinNowak**
- uptime can run slower on VMs

### CODE REVIEW

File: test/sysstats/main.d [hunk 1]
```diff
@@ -53,15 +54,19 @@ class MyApp : DaemonApp
...
+                    if (ProcVFS.getProcUptime() == uptime)
+                        return true;
                     stats = this.sys_stats.collect();
                     this.epoll.shutdown();
 
                     return false;
                 });
 
-        timer.set(0, 100, 0, 0);
+        timer.set(0, 10, 0, 10);
```
Comment (Burgos) [inline test/sysstats/main.d:69]: Ah, good idea.

**Review by Burgos — APPROVED** [review]

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by nemanja-boric-sociomantic** [comment]
Travis says No: 
```
./src/ocean/sys/stats/linux/ProcVFS.d(326): Error: fopen cannot be interpreted at compile time, because it has no available source code
```

**Comment by mihails-strasuns-sociomantic** [comment]
I'll take this over and push into v3.x.x branch, nevermind.

**Comment by mihails-strasuns-sociomantic** [comment]
https://github.com/sociomantic-tsunami/ocean/commit/8716d09afce1e8a1d91f2b2271bb32dc591f48aa

4.0.0-alpha.2 will follow in a moment

**Pull request closed without merging**